implementations/rust/ockam/ockam_transport_tcp/src/workers/sender.rs

```diff
@@ -177,6 +155,14 @@ impl Worker for TcpSendWorker {
                     return Err(TransportError::from(e).into());
                 }
             };
+
+            let keepalive = TcpKeepalive::new()
```

Comment:
Let's use following values for now:
tcp_keepalive_time = 300 (seconds)
tcp_keepalive_intvl = 75 (seconds)
tcp_keepalive_probes = 2 (number of probes)